spec/keytar-spec.js

```diff
@@ -0,0 +1,130 @@
+var assert = require('chai').assert
```

Comment:
What do you think about using async/await to increase this test's readability? We could include babel only in specs but still take advantage of the nicer syntax.